src/main/java/com/stripe/model/ExternalAccount.java

```diff
@@ -77,4 +91,8 @@ public DeletedExternalAccount delete(RequestOptions options) throws
 			APIConnectionException, CardException, APIException {
 		return request(RequestMethod.DELETE, this.getInstanceURL(), null, DeletedExternalAccount.class, options);
 	}
+
+	private String getValidateURL() {
```

Comment:
Kind of a nit: but putting the URL formatting in a new method like this seems a little unconventional relative to the rest of the library. What do you think about putting the `String.Format` [in-line as seen here](https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Invoice.java#L387,L388)?